gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -21,6 +21,7 @@ class ManageIQ::Providers::Amazon::CloudManager < ManageIQ::Providers::CloudMana
   require_nested :Vm
 
   OrchestrationTemplateCfn.register_eligible_manager(self)
+  ExtManagementSystem.register_cloud_discovery_type('amazon' => 'ec2')
```

Comment:
nope, I was blindly refactoring. Will have a look though, maybe we can get rid of this altogether. Good question!